tasks/profilingTasks.ts

```diff
@@ -0,0 +1,112 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import fs from 'fs';
+import * as gulp from 'gulp';
+import * as path from 'path';
+import { basicSlnTestProject, runIntegrationTest } from './testHelpers';
+import { outPath } from './projectPaths';
+import { execSync } from 'child_process';
+
+createProfilingTasks();
+
+function createProfilingTasks() {
+    const profilingOutputFolder = path.join(outPath, 'profiling');
+    gulp.task(`profiling:csharp:${basicSlnTestProject}`, async () => {
+        // Ensure the profiling output folder exists, otherwise the outputs will not get written.
+        fs.mkdirSync(path.join(outPath, 'profiling'), { recursive: true });
+
+        await runIntegrationTest(
```

Comment:
For now we just run the C# integration tests.  The devkit ones seem to be slightly less stable.  We can consider expanding this in the future.

Most of the data should be the same regardless.